lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1147,3 +1261,18 @@ def convert_to_utf8(input)
     end
   end
 end
+
+module Google
```

Comment:
Oh, this is intended. I added back in a later commit, since we are using `resource.dup` now. Without this patch, the resource is not duplicating labels (the new and the old resources point to the same labels...)

```
$ cat inner_map_not_dup.rb
require 'google/apis/logging_v2beta1'

old_resource = Google::Apis::LoggingV2beta1::MonitoredResource.new(
  labels: { 'aaa' => 'bbb' })
new_resource = old_resource.dup
new_resource.labels['aaa'] = 'ccc'
puts "new resource:"
puts new_resource.labels
puts "old resource:"
puts old_resource.labels

$ ruby inner_map_not_dup.rb
new resource:
{"aaa"=>"ccc"}
old resource:
{"aaa"=>"ccc"}
```

On a second thought, this should be a bug, right?